src/cordova/apple/OutlineAppleLib/Package.swift

```diff
@@ -6,6 +6,10 @@ import PackageDescription
 let package = Package(
     name: "OutlineAppleLib",
     defaultLocalization: "en",
+    platforms: [
+       .iOS(.v11),
```

Comment:
Add a comment that this is needed because of CocoaLumberjack.

What broke https://github.com/Jigsaw-Code/outline-client/issues/1800 is that CocoaLumberjack got upgraded to 3.8, and our package didn't restrict that.

I don't think we can upgrade this version without also upgrading the project.

Reply:
Done.